src/utils/getTotalMissedUpdate.ts

```diff
@@ -0,0 +1,16 @@
+export function getTotalMissedUpdates(date: Array<number>): number {
+    const currentStandupDate = new Date().setUTCHours(0, 0, 0, 0);
+    let lastStanupDate = 0;
+    let totalMissedUpdates = 0;
+    if (date.length === 0) {
+        return totalMissedUpdates;
+    }
+    for (let i = 0; i < date.length; i++) {
+        if (date[i] > lastStanupDate) {
+            lastStanupDate = date[i];
+        }
+    }
+    totalMissedUpdates =
+        Math.floor(currentStandupDate - lastStanupDate) / (24 * 60 * 60 * 1000);
```

Comment:
as per @TanishqSingla we have a moment library which will help us reduce the burden to calculate dates, Please try using that.

Reply:
here we are not returning dates we are just returning the difference between two dates and for that, we are using timeStamps from the /progresses?userId=${id}, and for that moment cannot be used.

Reply:
moment can be used to calculate difference between two dates

Reply:
fixed